Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM fedora:27
+
+RUN dnf install -y python2 python2-junit_xml beaker-client \
+	bison flex python2-mock dnf-plugins-core
+RUN dnf builddep -y kernel
+
+# Use a default checkout of the kernel as our workdir Also make it the default directory
+# and setup an environment variable to tell the skt executable to use this workdir
+#
+# Notice that we give 777 and 666 permissions on this directory. This is because Openshift
+# likes to run containers as random UIDs, and we cannot rely on SKT being root
+RUN git clone git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git /var/work && \
+	chmod -R ugo+w /var/work
```

Comment:
If you want to use this clone as `skt` workdir, then I think the proper way to do it would be using `skt` itself.
I.e. `skt --state merge --baserepo git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git` (provided we have the environment set and working directory created).

Otherwise we can have a bare clone as a local mirror which all the further `skt` commands would use as `--baserepo`, with `--workdir` someplace else. That would make creating new "workdirs" faster, but would need a bit more instructions, which might not be what we want.

Reply:
Good point. Will do.

Reply:
This ended up being problematic, as seen below. I don't think we should block the initial version of the container on this. If it is possible to do, we should experiment with it in a follow up.